File: app/src/main/java/com/sicsha/Main/MainActivity.java
package com.sicsha.Main;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import com.sicsha.R;

import butterknife.ButterKnife;

/**
 * Created by kiyoungLee on 2017-10-16.
 */

public class MainActivity extends AppCompatActivity{

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_act);
        ButterKnife.bind(this);

    }
}

File: app/src/main/java/com/sicsha/Main/MainActivity.kt
package com.sicsha.Main

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import com.sicsha.Main.Adapter.MainMenuTypeAdapter
import com.sicsha.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity(), MainContract.ActivityView {

    var presenter: MainContract.Presenter? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        presenter = MainPresenter()
        menu_type_list.layoutManager = LinearLayoutManager(this)
        presenter?.start()

    }

    override fun setAdapter(adapter: MainMenuTypeAdapter) {
        menu_type_list.adapter = adapter
    }
}
